src/client/app/components/HeaderButtonsComponent.tsx

```diff
@@ -102,16 +78,28 @@ export default class HeaderButtonsComponent extends React.Component<HeaderButton
 				<div className={this.props.showCollapsedMenuButton ? 'd-none d-lg-block' : ''}>
 					<TooltipHelpContainer page='all' />
 					<TooltipMarkerComponent page='all' helpTextId='help.home.header' />
-					<Link style={adminLinkStyle} to='/admin'><Button outline><FormattedMessage id='admin.panel' /></Button></Link>
-					<Link style={groupsLinkStyle} to='/groups'><Button outline><FormattedMessage id='groups' /></Button></Link>
-					<Link style={metersLinkStyle} to='/meters'><Button outline><FormattedMessage id='meters' /></Button></Link>
-					<Link style={mapsLinkStyle} to='/maps'><Button outline><FormattedMessage id='maps' /></Button></Link>
-					<Link style={csvLinkStyle} to='/csv'><Button outline><FormattedMessage id='csv' /></Button></Link>
-					<Link style={homeLinkStyle} to='/'><Button outline><FormattedMessage id='home' /></Button></Link>
+					<Link
```

Comment:
OED has not been completely consistent in how it formats items with multiple properties. Lately we have been placing them on separate lines as you did in your new lines. I think that is good but I also think we should be consistent within a file. Thus, I would reformat the other Link items to also be multiline.

Reply:
@huss I've reformated the link styles to be multilined. Does this suit the change you wanted?

Reply:
Thank you for the fast update. Everything looks good and about to be approved.